pages/services/beta-dcos-monitoring/0.5.0-beta/operations/prometheus/alert-rules/index.md

```diff
@@ -0,0 +1,156 @@
+
+<p class="message--note">
+<strong>NOTE:</strong> Mesosphere maintains a Git repository with Prometheus alerting rule configurations at `https://github.com/dcos/prometheus-alert-rules`.
+If you would like to use these preconfigured Prometheus alerting rules, you should set `prometheus.alert_rules_repository.config_repository.url` to `https://github.com/dcos/prometheus-alert-rules` and `prometheus.alert_rules_repository.config_repository.path` to `/rules`.
+As it is a public repository, there is no need to set up the `credentials`.
+</p>
+
+```json
+{
+  "prometheus": {
+    "alert_rules_repository": {
+      "config_repository": {
+        "url": "https://github.com/dcos/prometheus-alert-rules",
+        "path": "/rules"
+      }
+    }
+  }
+}
+```
+
+## Create secrets for Git repository credentials
+
+If the Git repository containing the alert rules is private, you will need to configure the secrets first.
+Currently, the following Auth types are supported.
+
+### HTTP Auth
+
+```bash
+dcos security secrets create --value=<GITHUB_USERNAME> githubusername
+dcos security secrets create --value=<GITHUB_PASSWORD> githubpassword
+```
+
+For Github, the password can be the API token.
+
+Create a custom option file (`options.json`) like the following.
+You may omit the `credentials` section if the Git repository is public.
+
+```json
+{
+  "prometheus": {
+    "alert_rules_repository": {
+      "url": "https://github.com/company/alert-rules",
+      "path": "/production",
+      "credentials": {
+        "username": "githubusername",
+        "password": "githubpassword"
+      }
+    }
+  }
+}
+```
+
+### SSH Auth
+
+```bash
+dcos security secrets create -f <PATH_TO_PRIVATE_KEY> gitsshkey
+```
+
+For Github, please make sure to add [Deployment Key](https://developer.github.com/v3/guides/managing-deploy-keys/#deploy-keys) (i.e., the public key) to the repository.
+
+Create a custom option file (`options.json`) like the following.
+
+```json
+{
+  "prometheus": {
+    "alert_rules_repository": {
+      "url": "git@github.com:company/alert-rules.git",
+      "path": "/production",
+      "credentials": {
+        "deploy_key": "gitsshkey"
+      }
+    }
+  }
+}
+```
+
+Note that you'll have to use `git@github.com:<USER>/<REPO>.git` instead of `https` as the scheme of the URL.
+
+## Fetching from a branch in a Git repository
+
+By default, the service will fetch from the master branch (that is, `refs/heads/master`) of the Git repository.
+
+If you want to fetch the alert rules from another branch in a Git repository, you can set the `reference_name` field:
+
+```json
+{
+  "prometheus": {
+    "alert_rules_repository": {
+      "url": "https://github.com/company/alert-rules",
+      "path": "/production",
+      "reference_name": "refs/heads/my_branch"
+    }
+  }
+}
+
+```
+
+## Install {{ model.techName }} service
+
+Then, install the service:
```

Comment:
Rewrite line #141 with the following:
Install the service using the following command.

Reply:
@anithamuthu79 As I mentioned in the comment, I would prefer not to make language changes such as these, as these docs will be removed once we release v1.0. I have made most of these language changes already in that version of docs (we've done a round of doc reviews where you have already requested these same changes, but they were done in a later version), so I would prefer to not have to duplicate those changes now, if that's okay, just to get this release out the door.

There will be one final review of dcos-monitoring v1.0 docs here on this repo prior to the release, so I would rather make the changes there instead of here (this version will not be merged back into the newer 1.0).